test/uk/gov/hmrc/cataloguefrontend/FrontendRouteWarningsPageSpec.scala

```diff
@@ -17,15 +17,13 @@
 package uk.gov.hmrc.cataloguefrontend
 
 import com.github.tomakehurst.wiremock.http.RequestMethod._
-import play.api.libs.ws._
 import uk.gov.hmrc.cataloguefrontend.util.UnitSpec
 
 class FrontendRouteWarningsPageSpec extends UnitSpec with FakeApplicationBuilder {
 
-  private[this] lazy val ws = app.injector.instanceOf[WSClient]
-
   override def beforeEach(): Unit = {
     super.beforeEach()
+    serviceEndpoint(POST, "/internal-auth/auth"                                          , willRespondWith = (200, Some("""{"retrievals": []}""")))
```

Comment:
worth moving this into `WireMockEndpoints`? It will be needed in most tests

Reply:
Good idea, done.